ibis/backends/trino/datatypes.py

```diff
@@ -93,7 +93,7 @@ def parse(text: str, default_decimal_parameters=(18, 3)) -> dt.DataType:
     )
 
     primitive = (
-        spaceless_string("interval").result(dt.Interval())
+        spaceless_string("interval").result(dt.Interval(unit='s'))
```

Comment:
I'm not sure `s` is correct here, but I don't know if there's a reasonable default since Trino has two kinds of interval types.